vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -206,16 +205,13 @@ type SecureServingInfo struct {
 	// Listener is the secure server network listener.
 	Listener net.Listener
 
-	// Cert is the main server cert which is used if SNI does not match. Cert must be non-nil and is
-	// allowed to be in SNICerts.
-	Cert *tls.Certificate
-
-	// SNICerts are the TLS certificates by name used for SNI.
-	SNICerts map[string]*tls.Certificate
-
 	// ClientCA is the certificate bundle for all the signers that you'll recognize for incoming client certificates
 	ClientCA *x509.CertPool
 
+	// DynamicCertificates is an alternative to Cert and SNICerts that is limited to files, but those file will
+	// be monitored for changes every minute and allows the certificates to be updated dynamically.
+	DynamicCertificates *DynamicCertificateConfig
```

Comment:
why alternative if Cert and SNICerts are removed?

Reply:
> why alternative if Cert and SNICerts are removed?

just old doc.  I didn't realize when I started that it was safe to remove them.